packages/terra-worklist-data-grid/tests/wdio/worklist-data-grid-spec.js

```diff
@@ -184,6 +184,18 @@ Terra.describeViewports('WorklistDataGrid', ['medium', 'large'], () => {
 
       Terra.validates.element('cell-3-1-focused', { selector: defaultSelector });
     });
+
+    it('hovers over selectable cell on an even row', () => {
+      browser.$$('tbody tr')[0].$$('td')[0].moveTo();
+      browser.pause(2000);
```

Comment:
Curious if instead of these long pauses we could use some element selector with `.waitForDisplayed()` or a `waitUntil` condition.

Reply:
The delay is to ensure time for the hover action to trigger.  There is nothing to be displayed.